airflow/operators/python.py

```diff
@@ -252,13 +268,21 @@ def execute(self, context: Context) -> Any:
             self.log.info('Proceeding with downstream tasks...')
             return condition
 
-        self.log.info('Skipping downstream tasks...')
-
-        downstream_tasks = context["task"].get_flat_relatives(upstream=False)
-        self.log.debug("Downstream task_ids %s", downstream_tasks)
+        downstream_tasks = context['task'].get_flat_relatives(upstream=False)
+        self.log.debug("Downstream tasks %s", downstream_tasks)
```

Comment:
The previous log (task _ID_) seems more accurate? Doesn’t really matter though.

Reply:
I was thinking that most users refer to tasks by their `task_id` (and a little that `task_ids` isn't _actually_ an attribute). Maybe a happy medium of `self.log.debug("Downstream task IDs %s", downstream_tasks)` works?